plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1493,6 +1537,14 @@ public Set<HivePrivilegeInfo> listTablePrivileges(String databaseName, String ta
         }
     }
 
+    @Override
+    public boolean supportBatchGetViews()
+    {
+        // When this is disabled, views are tables with a specific flag,
+        // and there is no way to fetch its names all at once
+        return translateHiveViews;
```

Comment:
That disables the improvement for Delta Lake connector. And for Hive connector under default configuration (`io.trino.plugin.hive.HiveConfig#translateHiveViews` defaults to false). Which sounds like it kills the point?

Reply:
I understand this will optimize only a narrow path. However, having a hive connector with translated views enabled is not rare. Many BI tools execute `select * from informations_schema.tables` at the beginning and this change makes it faster by orders of magnitude.